src/core/contracts/module-manager.ts

```diff
@@ -0,0 +1,14 @@
+import { CommandMeta, CommandModule, CommandModuleDefs, Module } from '../../types/core-modules';
```

Comment:
shouldn't we import types like `import type`?

Reply:
We should for cleanliness, not adding it doesnt seem to affect bundle output though